lib/redis/pipeline.rb

```diff
@@ -86,7 +86,13 @@ def finish(replies)
       end
 
       def commands
-        [[:multi]] + super + [[:exec]]
+        a = super
```

Comment:
If there are no `@futures`, we needn't `super` either, huh! Would be nice to have e.g. `Redis::Pipeline#empty?` which delegates to `@futures.empty?` (or `none?`/`any?`…), leaving us with a concise
```ruby
def commands
  if empty?
    []
  else
    [[:multi]] + super + [[:exec]]
  end
end
```